src/__tests__/modals/HelpModal.test.tsx

```diff
@@ -2,15 +2,15 @@ import { describe, expect, it } from '@jest/globals'
 import * as React from 'react'
 import { createRenderer } from 'react-test-renderer/shallow'
 
-import { HelpModalComponent } from '../../components/modals/HelpModal'
-import { getTheme } from '../../components/services/ThemeContext'
+import { HelpModal } from '../../components/modals/HelpModal'
 import { fakeAirshipBridge } from '../../util/fake/fakeAirshipBridge'
+import { fakeNavigation } from '../../util/fake/fakeSceneProps'
 
 describe('HelpModal', () => {
   it('should render with loading props', () => {
     const renderer = createRenderer()
 
-    const actual = renderer.render(<HelpModalComponent bridge={fakeAirshipBridge} theme={getTheme()} />)
+    const actual = renderer.render(<HelpModal bridge={fakeAirshipBridge} navigation={fakeNavigation} />)
```

Comment:
We are using the shallow renderer on this scene. We need to upgrade to the deep renderer, so the snapshot diffs will actually contain something useful:

```ts
import TestRenderer from 'react-test-renderer'

    const renderer = TestRenderer.create(
      <FakeProviders>
        <HelpModal ... />
      </FakeProviders>
    )

    expect(renderer.toJSON()).toMatchSnapshot()
    renderer.unmount()
```